.ci_support/linux_64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11.yaml

```diff
@@ -32,7 +32,11 @@ pin_run_as_build:
     min_pin: x.x
     max_pin: x.x
 python:
+- 3.10.* *_cpython
+- 3.11.* *_cpython
 - 3.12.* *_cpython
+- 3.8.* *_cpython
+- 3.9.* *_cpython
```

Comment:
Note that we now get a build matrix over `python` without the top-level `noarch`

Meaning that we rebuild each package using `python` 5 times (even though each package is identical)

Reply:
Reverted